docs/tutorials/tabular_prediction/tabular-gpu.md

```diff
@@ -41,19 +41,47 @@ hyperparameters
 
 ### Text models
 
-Text model preset to use can be set via:
+Text model's configuration can be set via:
 
 ```{.python}
-hyperparameters['AG_TEXT_NN'] = ['<preset>']
+hyperparameters['AG_TEXT_NN'] = {
+    "model": "/path/to/model/yaml/file",
+    "data": "/path/to/data/yaml/file",
+    "optimization": "/path/to/optimization/yaml/file",
+    "environment": "/path/to/optimization/yaml/file",
+}
 ```
 
-Available text model presets:
+
+Text model uses the following configurations as default.
+
 
 ```{.python .input}
-from autogluon.text.text_prediction.presets import ag_text_presets
-ag_text_presets.list_keys()
+from omegaconf import OmegaConf
+from autogluon.text.automm.utils import get_config
+config = get_config({
+    "model": "fusion_mlp_text_tabular",
+    "data": "default",
+    "optimization": "adamw",
+    "environment": "default",
+})
+OmegaConf.to_container(config)
```

Comment:
This would need to be changed if the default ever changed, consider instead creating a method that returns whatever the default config is (very similar if not the same as the presets logic)

Reply:
Agree.

Reply:
Preset is supported now.